gap/aut-func.gi

```diff
@@ -887,79 +887,133 @@ end);
 ##  expressions as arguments
 ##
 InstallGlobalFunction(IntersectionLanguage, function(a1,a2)
-    local   HashPair,  ht,  init,  states,  m,  i,  t1,  t2,  t,  st,  a,  
-            nst,  he,  finals,  p,  q;
+    local  HashPair, ht, init, states, m, s1, s2, t1, t2, t, i, st, a, x, y, nst, he,   finals, p, q, numofinitst,m1,m2;
```

Comment:
It seems this whole function is never invoked by any test in the test suite. Perhaps you could add one?

Reply:
This is now tested. Checking how many of the cases/lines are.